applications/connecting_applications_to_services/odc-connecting-an-application-to-a-service-using-the-developer-perspective.adoc

```diff
@@ -19,7 +20,7 @@ This procedure walks you through an example of creating a visual connection betw
 .Prerequisites
 
 * Ensure that you have created and deployed a Spring PetClinic sample application using the *Developer* perspective.
-* Ensure that you have created and deployed a Crunchy PostgreSQL database instance using the *Developer* perspective. This instance has the following three components: `hippo-backup`, `hippo-instance`, and  `hippo-pgbouncer`.
+* Ensure that you have created and deployed a Crunchy PostgreSQL database instance using the *Developer* perspective. This instance has the following three components: `hippo-backup`, `hippo-instance`, `hippo-repo-host`, and  `hippo-pgbouncer`.
```

Comment:
Would it make sense to start prerequisites with "You have", like in the preceding file.

Reply:
Not 3 components I like suggestion from @rolfedh below